src/Utility/Xml.php

```diff
@@ -265,7 +265,7 @@ protected static function load(string $input, array $options, Closure $callable)
      */
     public static function fromArray(object|array $input, array $options = []): SimpleXMLElement|DOMDocument
     {
-        if (is_object($input) && method_exists($input, 'toArray') && is_callable([$input, 'toArray'])) {
+        if (is_object($input) && is_callable([$input, 'toArray'])) {
```

Comment:
I was thinking if we either want to make sure the method_exists OR if its callable... so I chose to just check if its callable.

Reply:
If someone implements `__call` they could be in for a surprise, but also why do that with class that goes into code like `Xml` :thinking: